docs/Over_OSPO-NL.md

```diff
@@ -1,7 +1,29 @@
-```{warning}
-De documentatie is nog volop in ontwikkeling
-```
-
 # Over OSPO-NL
  
-TO DO
\ No newline at end of file
+OSPO staat voor Open Source Program Office.
+De toevoeging NL doelt op de Nederlandse samenwerking om kennis en best practices te delen
+voor het opzetten en uitvoeren van OSPO's (of OSPOs) bij organisaties in Nederland.
+Het is niet van belang of deze organisatie commerciële bedrijven zijn of overheidsinstanties.
+
+Het initiatief van deze community en samenwerking is ontstaan naar aanleiding van de
+[OSPOlogy.live](https://community.linuxfoundation.org/events/details/lfhq-ospology-european-chapter-presents-ospologylive-share-learn-netherlands/)
+van januari 2023.
+Dit event werd georganiseerd door de [Linux Foundation](https://www.linuxfoundation.org/) en gehost door [Alliander](https://www.alliander.com/en/open-source/projects/).
+
+OSPO-NL is een open community welke gestart is door [Alliander](https://www.alliander.com/en/open-source/projects/),

```

Comment:
@Jonasvdbo en @nicorikken , is dit een goede URL voor op deze pagina voor Alliander?

Reply:
Ja top!